src/rustdoc/meetings.md

```diff
@@ -0,0 +1,13 @@
+# Meetings
+
+The rustdoc team hosts a meeting every second Monday on each month on the 
+[t-rustdoc/meetings channel on zulip](https://rust-lang.zulipchat.com/#narrow/channel/393423-t-rustdoc.2Fmeetings)
+at 21:00 CET (UTC+1) and CEST on summer (UTC+2).
+
+A new thread is open a month ahead of time to remind people who want to attend about the time
+and the agenda.
+
+Anyone is open to participate to the meetings.
+
+If you want to add items to the agenda, please comment on the Zulip thread for the next meeting
+about the items you want to see discussed. A member of the team will then add it to the agenda.
```

Comment:
This policy is a bit ambiguous

1. the wording partially suggests that team members should also follow this process instead of just adding an item directly
2. it doesn't actually make it clear what purpose this indirection serves.  is it filtering out spam? delaying agenda items to future meetings if the current meeting's agenda is already full?

Reply:
Indeed, gonna clarify it.